### PR TITLE
Adds CI to verify maps are within 255x255x1

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
With MILLA on the horizon and maps being required to be 255x255 or less, this feels like a good way to get out ahead. Also we almost certainly have some crap coded to rely on 255x255 maps.

## Why It's Good For The Game
Sanity good.

## Images of changes
N/A

## Testing
Ran the script with and without error cases, worked on both.

## Changelog
NPFC

### CODE REVIEW

**Comment by S34NW** [comment]
Even without the technical limitations of MILLA, I think this is a good idea. We don't want people making out of control size maps because they see perf is super good.

**Comment by AffectedArc07** [comment]
> Even without the technical limitations of MILLA, I think this is a good idea. We don't want people making out of control size maps because they see perf is super good.

True dat

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -92,7 +93,13 @@ jobs:
...
+        maptype:
+          [
+            '/datum/map/boxstation',
+            '/datum/map/deltastation',
+            '/datum/map/metastation',
+            '/datum/map/cerestation',
+          ]
         byondtype: ['STABLE']
     services:
       mariadb:
```
Comment (AffectedArc07) [inline .github/workflows/ci.yml:105]: FUCK AUTOFORMAT

**Comment by FunnyMan3595** [comment]
> With MILLA on the horizon and maps being required to be 255x255 or less, this feels like a good way to get out ahead. Also we almost certainly have some crap coded to rely on 255x255 maps.

For what it's worth, this isn't a hard requirement for MILLA, just how it's currently written. Increasing the cap uniformly would be easy (it's a constant), and making it settable in `initialize(z_level)` is also possible (though Rust might take some convincing).

**Review by Burzah — APPROVED** [review]

**Comment by warriorstar-orion** [comment]
Could you wrap the script in a timer? I like to have a handle on overall CI run times in case things ever get out of hand. Should be as straightforward as
```py
import time

def main():
    start = time.time()
    # do the thing
    end = time.time()
    print(f"\ncheck_map_sizes.py completed in {end - start:.2f}s\n")
```

**Review by Burzah — COMMENTED** [review]
I am not sure if Warrior's comment got registered as a review comment, but I will echo it just in case.

```Could you wrap the script in a timer? I like to have a handle on overall CI run times in case things ever get out of hand. Should be as straightforward as```

```py
import time

def main():
    start = time.time()
    # do the thing
    end = time.time()
    print(f"\ncheck_map_sizes.py completed in {end - start:.2f}s\n")

**Comment by AffectedArc07** [comment]
image

**Pull request merged**